misk-hibernate-testing/src/main/kotlin/misk/jdbc/VitessScaleSafetyChecks.kt

```diff
@@ -473,18 +474,18 @@ class VitessScaleSafetyChecks(
     }
   }
 
-  inner class CowriteDetector : ExtendedQueryExectionListener() {
-    private val keyspaceIdsThisTransaction: ThreadLocal<LinkedHashSet<String>> =
-        ThreadLocal.withInitial { LinkedHashSet<String>() }
+  inner class CowriteDetector : ExtendedQueryExecutionListener() {
+
+    private val vtgateKeyspaceIdRegex = "vtgate:: keyspace_id:([^ ]+)".toRegex()
```

Comment:
I think this should be in the companion object so we don't have to parse the regexp every time we start a transaction.

Reply:
Done